.github/workflows/main.yml

```diff
@@ -78,17 +78,34 @@ jobs:
 
   build-freebsd:
     runs-on: macos-latest
+    name: build-freebsd-package
     steps:
     - name: Checkout
       uses: actions/checkout@v2
+    - name: Find tag
+      id: tagger
+      uses: jimschubert/query-tag-action@v1
     - name: Build
       id: test
       uses: vmactions/freebsd-vm@v0.1.4
       with:
         usesh: true
         mem: 8192
         prepare: |
-          pkg install -y git gtar python3 glib meson pkgconf gobject-introspection \
-            json-glib gnutls gtk-doc vala sqlite3 curl gcab libarchive libelf \
-            gettext-tools gnutls usbids polkit
-        run: ./contrib/ci/build_freebsd.sh
```

Comment:
maybe we should drop this file too as part of this PR.

Reply:
Deleted.